randovania/interface_common/options.py

```diff
@@ -459,6 +478,12 @@ def hidden_preset_uuids(self):
     def hidden_preset_uuids(self, value):
         self._edit_field("hidden_preset_uuids", value)
 
+    def get_preset_order_for(self, game: RandovaniaGame) -> list[uuid.UUID]:
+        return list(_return_with_default(getattr(self, f"preset_order_{game.value}"), list))
```

Comment:
Isn't `list` uninitialized here?

Reply:
You meant the second argument to `_return_with_default`? That function receives a callable that creates a default value. By passing `list` it can create a list when `preset_order_...` is None.